tests/test_custom_models.py

```diff
@@ -445,6 +489,10 @@ def test_only_params_are_updated(self, test_name, model_id, config_cls, config_k
         params_after = dict(model.named_parameters())
         self.assertEqual(params_before.keys(), params_after.keys())
 
+        if isinstance(model, ModelConv2D):
```

Comment:
Probably some debug code, please remove.

Reply:
You are right, sorry for that.